tests/core/input.cddl

```diff
@@ -59,21 +59,21 @@ i64 = int .size 8 ; 8 bytes
 
 signed_ints = [
 	u_8: uint .size 1,
-	u_16: uint .size 2,
-	u_32: uint .size 4,
+	u_16: uint .le 65535,
+	u_32: 0..4294967295,
 	u_64: uint .size 8,
-	i_8: int .size 1,
+	i_8: -128..127,
```

Comment:
could we keep the old test along with the new one?

Reply:
@gostkin is this good enough coverage now? to get 100% we'd have to have more structs to test `.le`, `.size` and `n..m` for all cases but this should at least improve it, and is more coverage than we previously had before this PR. The code mostly follows the same path so I don't think we'd gain much from that either.